### ISSUE TITLE
ATOM should check for chains for the necessary tf's only

### ISSUE THREAD

**Issue by brunofavs** [lardemua/atom#933]
Right now data collector checks if ALL tf's have a connection to world.

This crashes the node whenever working with robots with multiple isolated branches.

The verification should be more selective.

**Comment by miguelriemoliveira** [comment]
Right, a big change. Will be in the configure, datacollector and others.
